extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -47,6 +49,9 @@ export class PlatformStore {
     /** @private {?Promise<!Array<!./entitlement.Entitlement>>} */
     this.allResolvedPromise_ = null;
 
+    /** @private {!Array<string>} */
+    this.failedPlatforms_ = [];
```

Comment:
Just occurred to me. When you are doing "allResolved", shouldn't you now be doing `entitlements_.length + failedPlatforms_.length`?

Reply:
Not really, but yeah I see why this can be confusing, hence moving code around a bit to make it more readable